CITATIONS.md

```diff
@@ -1,24 +1,49 @@
-# arcadia-metagenomic-assembly-nf Citations
+# Arcadia-Science/metagenomics Citations
 
-## [nf-core](https://pubmed.ncbi.nlm.nih.gov/32055031/)
+Below are citations of tools used in the pipeline that you should cite in your work when using this pipeline.
+
+## Nextflow and Reporting Tools
+
+- [nf-core](https://pubmed.ncbi.nlm.nih.gov/32055031/)
 
 > Ewels PA, Peltzer A, Fillinger S, Patel H, Alneberg J, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol. 2020 Mar;38(3):276-278. doi: 10.1038/s41587-020-0439-x. PubMed PMID: 32055031.
```

Comment:
can you switch this to the arcadia citation style? you can copy and paste some of them from here if you want: https://github.com/Arcadia-Science/seqqc/blob/master/CITATIONS.md

Reply:
I will create an issue for this and address later